contracts/TreasuryManager.sol

```diff
@@ -5,47 +5,49 @@ import "@openzeppelin/contracts/proxy/utils/Initializable.sol";
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
```

Comment:
Ok changes look good, can you re-arrange the functions so the `onlyOwner` functions are at the top, the `onlyManager` functions are next and then the private  / internal functions are at the bottom?